spec/controllers/instruments_controller_spec.rb

```diff
@@ -275,7 +275,7 @@
                                       password: "password",
                                       type: RelaySynaccessRevA.name,
                                       auto_logout: true,
-                                      auto_logout_minutes: 15,
```

Comment:
There are [some specs ](https://github.com/tablexi/nucore-open/blob/f22ff0b945db10cbd1c866ead2446114e12043b5/spec/controllers/instruments_controller_spec.rb#L267)in this file that describe controller logic that I don't think we're using any more.
Can you please check if that's the case and if so, move the test logic from this file into the new system spec?